schemas/v3.1/schema.yaml

```diff
@@ -0,0 +1,810 @@
+                  minItems: 1
+                default:
+                  type: string
+              unevaluatedProperties: false
+        unevaluatedProperties: false
+      Response:
+        $anchor: Response
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - description
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              headers:
+                type: object
+                additionalProperties:
+                  $ref: '#Header'
+              content:
+                type: object
+                additionalProperties:
+                  $ref: '#MediaType'
+              links:
+                type: object
+                additionalProperteis:
+                  $ref: '#Link'
+            unevaluatedProperties: false
+      MediaType:
+        $anchor: MediaType
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+        properties:
+          encoding:
+            type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/StyledFormComplexNoDefaultOrEmpty'
+            properties:
+              contentType:
+                type: string
+              headers:
+                type: object
+                additionalProperties:
+                  $ref: '#Header'
+        unevaluatedProperties: false
+      Example:
+        $anchor: Example
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/Describable'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            properties:
+              summary:
+                type: string
+              value: true
+              externalValue:
+                type: string
+                format: uri-reference
+            unevaluatedProperties: false
+      Header:
+        $anchor: Header
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            required:
+              - schema
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/DescReqDep'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+            oneOf:
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithSchemaAndExamples'
+                  - $ref: '#/$defs/Mixins/$defs/StyledSimple'
+              - allOf:
+                  - $ref: '#/$defs/Mixins/$defs/WithContent'
+                  - $ref: '#/$defs/Mixins/$defs/WithEmpty'
+            unevaluatedProperties: false
+      Paths:
+        $anchor: Paths
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        patternProperties:
+          ^\/:
+            type: object
+            oneOf:
+              - $ref: '#Reference'
+              - $ref: '#PathItem'
+        unevaluatedProperties: false
+      PathItem:
```

Comment:
Does this `PathItem` schema deliberately allow additional/unevaluated properties? Should it not be constrained to `get`, `post`, `put`, `patch`, `delete`, `head`, `options`, `trace`?

Reply:
I think there was a github issue raised recently asking why the spec should be constrained to the current list of HTTP methods, rather than being open to new ones being added in the future. I would lean towards being open -- the `examples` keyword could be used to list the common/expected values as a hint to humans.